sdk/eventhub/Azure.Messaging.EventHubs/src/EventProcessorClient.cs

```diff
@@ -54,7 +54,7 @@ public class EventProcessorClient : IAsyncDisposable
         ///
         public Func<InitializePartitionProcessingContext, Task> InitializeProcessingForPartitionAsync
         {
-            internal get => _initializeProcessingForPartitionAsync;
+            protected get => _initializeProcessingForPartitionAsync;
```

Comment:
I agree that's pretty useless to protect it. Just thought it wouldn't make much sense to let them call it directly from the `EventProcessorClient` because the handlers are not intended to be used like that. We can make it `public`.